sdk/storage/storage-blob/test/aborter.spec.ts

```diff
@@ -26,67 +26,63 @@ describe("Aborter", () => {
     recorder.stop();
   });
 
-  it("Should abort after aborter timeout", async () => {
-    recorder.skip(
-      "browser",
-      "Abort: browser testing unexpectedly finishes when a request is aborted during playback, shortcomings of `nise` library"
-    );
-    try {
-      await containerClient.create({ abortSignal: AbortController.timeout(1) });
-      assert.fail();
-    } catch (err) {
-      assert.equal(err.name, "AbortError");
-      assert.equal(err.message, "The operation was aborted.", "Unexpected error caught: " + err);
-    }
+  it("Should abort after aborter timeout", function() {
+    async () => {
```

Comment:
This is a anonymouse async method definition, but this method doesn't get executed.

Reply:
Yeah, missed it somehow  😌. Closing the PR.